src/types.ts

```diff
@@ -121,6 +136,16 @@ export interface PoolWithStats extends Pool {
     sevenDayProtocolFeesGeneratedInEth: number;
 }
 
+export interface PoolWithHistoricalStats extends Pool {
+    allTimeStats: {
+        operatorRewardInEth: number;
+        membersRewardInEth: number;
+        totalRewardInEth: number;
+        protocolFeesGeneratedInEth: number;
```

Comment:
I feel like we should figure out how to make this fit in with `AllTimeStakingStats` (returned by the /stats endpoint) and `AllTimePoolRewards`. Can't they be the same thing?

Reply:
Good point, they should be the same. My bad 😳